flash/tabular/forecasting/data.py

```diff
@@ -155,12 +157,24 @@ def from_data_frame(
         num_workers: Optional[int] = None,
         **preprocess_kwargs: Any,
     ):
-        """Creates a :class:`~flash.tabular.data.TabularClassificationData` object from the given data frames.
+        """Creates a :class:`~flash.tabular.forecasting.data.TabularForecastingData` object from the given data
+        frames.
+
+        .. note::
+
+            The ``time_idx``, ``target``, and ``group_ids`` do not need to be provided if ``parameters`` are passed
+            instead. These can be obtained from the
+            :attr:`~flash.tabular.forecasting.data.TabularForecastingData.parameters` attribute of the
+            :class:`~flash.tabular.forecasting.data.TabularForecastingData` object that contains your training data.
 
         Args:
-            group_ids:
-            target:
             time_idx:
+            target: Column denoting the target or list of columns denoting the target.
```

Comment:
thinking of including `_col` in the name to be specific that it column name

Reply:
Good suggestion, will do in a seperate PR :smiley: